scripts/scheduled-leaderboard-purge.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Set all leaderboard records to inactive, where no game was played in the last 3 months
+SCRIPT_BASE_DIR=$(cd "$(dirname "$0")" || exit; cd ../; pwd)
+DB_CONFIG_FILE="$SCRIPT_BASE_DIR/config/faf-db/faf-db.env"
+
+echo "Reading db config from $DB_CONFIG_FILE"
+
+# shellcheck source=../config/faf-db/faf-db.env"
+source "$DB_CONFIG_FILE"
+
+# Ladder1v1 leaderboard
+echo "Processing inactive users for ladder1v1 leaderboard"
+
+docker exec -i faf-db mysql -D "${MYSQL_DATABASE}" <<SQL_SCRIPT
+  CREATE TEMPORARY TABLE active_players AS
```

Comment:
Question is: do we need that temporary table? 🙈
We could join on that query as it is and no need to have temporary table.

Reply:
Probably no, but my experience with MySQL is: if you process large datasets and only need a small result, put it into a temporary table. Also makes analysing easier.